debug/test/browser/devtools.test.js

```diff
@@ -70,23 +74,23 @@ function createMockHook() {
 
 /**
  * Check if the event has been seen (=mounted in most cases) before.
- * @param {import('../../src/internal').VNode} event
- * @param {Set<import('../../src/internal').VNode>} seen
+ * @param {VNode} vnode
+ * @param {Set<VNode>} seen
  * @returns {boolean}
  */
 function checkPreceding(vnode, seen) {
 	if (vnode==null) return true;
 
 	// If a leaf node is a Fragment and has no children it will be skipped
-	if (vnode.type===Fragment && vnode._children==0) return true;
+	if (vnode.type===Fragment && (vnode._children==null || vnode._children.length===0)) return true;
```

Comment:
Found an actual bug in the test assertions here: `vnode._children` is never of type `number` :tada: